src/pocketmine/block/BlockIds.php

```diff
@@ -142,18 +142,18 @@
 	const ENCHANTING_TABLE = 116, ENCHANT_TABLE = 116, ENCHANTMENT_TABLE = 116;
 	const BREWING_STAND_BLOCK = 117;
 	const CAULDRON_BLOCK = 118;
-
-	const END_PORTAL_FRAME = 120, END_PORTAL = 120;
+	//const END_PORTAL = 119; //Confirmed
```

Comment:
Why don't we leave it to be added when it is ready? And what is the purpose of `//Confirmed`?